R/Functions_L1_Pop.R

```diff
@@ -1131,23 +1129,36 @@ pullVirginQueens <- function(x, nInd = NULL, use = "rand", collapse = FALSE) {
 #'   (\code{\link{Colony-class}}) or  (\code{\link{MultiColony-class}}), one
 #'   virgin queens is selected at random, mated, and promoted to the queen of
 #'   the colony. Other virgin queens are destroyed. Mated drones (fathers) are
-#'   stored for producing progeny at a later stage.
+#'   stored for producing progeny at a later stage. Since this is a complex
```

Comment:
Remove this part since now the function is not that complex (well not as much as you thought)